src/System.Security.Cryptography.Encoding/tests/Asn1/Writer/WriteInteger.cs

```diff
@@ -401,5 +401,99 @@ public void VerifyWriteInteger_ConstructedIgnored(PublicEncodingRules ruleSet)
                 Verify(writer, "020100800100020100800100020100800100");
             }
         }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public static void WriteAfterDispose(bool empty)
+        {
+            using (AsnWriter writer = new AsnWriter(AsnEncodingRules.DER))
+            {
+                if (!empty)
+                {
+                    writer.WriteNull();
+                }
+
+                writer.Dispose();
+
+                Assert.Throws<ObjectDisposedException>(
+                    () => writer.WriteInteger(1));
+
+                Assert.Throws<ObjectDisposedException>(
+                    () => writer.WriteInteger(1UL));
+
+                Assert.Throws<ObjectDisposedException>(
+                    () => writer.WriteInteger(BigInteger.One));
+
+#if netcoreapp
+                Assert.Throws<ObjectDisposedException>(
+                    () => writer.WriteInteger(BigInteger.One.ToByteArray(isBigEndian: true)));
```

Comment:
Since 1 is { 1 } in both big and little endian, changing the test to not specify big endian seems better than conditional compilation.